src/lib/journey/config.store.ts

```diff
@@ -43,27 +43,25 @@ const defaultJourneys = {
 // Ensure default follows schema
 journeyConfigSchema.parse(defaultJourneys);
 
-export let configuredJourneys: Readable<StoreItem[]>;
+export let configuredJourneysStore: Writable<StoreItem[]> = writable(
+  (Object.keys(defaultJourneys) as JourneyKeys[]).map((key) => ({
+    ...defaultJourneys[key],
+    key,
+  })),
+);
```

Comment:
You'll see this throughout this PR. I've pulled the store creation out of the `initialize` function. I initially wanted to hide the store and only expose the stores in a narrowed way, but I discovered that it limited the flexibility of the framework too much. Having the store exported like this means references don't have to be passed around so much.